opencti-platform/opencti-front/src/private/components/data/connectors/ConnectorWorks.tsx

```diff
@@ -0,0 +1,388 @@
+  options: Options[]
+  relay: RelayRefetchProp
+}
+
+const ConnectorWorksComponent: FunctionComponent<ConnectorWorksComponentProps> = ({ data, options, relay }) => {
+  // const works: NonNullable<ConnectorWorks_data$data['works']>['edges'] = pathOr([], ['works', 'edges'], data);
+  const works = data.works?.edges ?? [];
+  const { t_i18n, nsdt } = useFormatter();
+  const classes = useStyles();
+  const [displayErrors, setDisplayErrors] = useState<boolean>(false);
+  const [errors, setErrors] = useState<WorkMessages>([]);
+
+  const handleOpenErrors = (errorsList: WorkMessages | null) => {
+    if (errorsList !== null) {
+      setDisplayErrors(true);
+      setErrors(errorsList);
+    }
+  };
+
+  const handleCloseErrors = () => {
+    setDisplayErrors(false);
+    setErrors([]);
+  };
+
+  // eslint-disable-next-line class-methods-use-this
+  const handleDeleteWork = (workId: string) => {
+    commitMutation({
+      mutation: connectorWorksWorkDeletionMutation,
+      variables: {
+        id: workId,
+      },
+      updater: undefined,
+      optimisticUpdater: undefined,
+      optimisticResponse: undefined,
+      onCompleted: () => {
+        MESSAGING$.notifySuccess('The work has been deleted');
+      },
+      onError: undefined,
+      setSubmitting: undefined,
+    });
+  };
+
+  useEffect(() => {
+    const subscription = interval$.subscribe(() => {
+      relay.refetch(options);
+    });
+    return () => subscription.unsubscribe();
+  }, []);
+
+  return (
+    <div>
+      {works.length === 0 && (
+        <Paper
+          classes={{ root: classes.paper }}
+          variant="outlined"
+          style={{ marginBottom: 20 }}
+        >
+          <div
+            style={{
+              display: 'table',
+              height: '100%',
+              width: '100%',
+            }}
+          >
+            <span
+              style={{
+                display: 'table-cell',
+                verticalAlign: 'middle',
+                textAlign: 'center',
+              }}
+            >
+              {t_i18n('No work')}
+            </span>
+          </div>
+        </Paper>
+      )}
+      {works.map((workEdge) => {
+        const work = workEdge?.node;
+        if (!work) return null;
+        const { tracking } = work;
+        return (
+          <Paper
+            key={work.id}
+            classes={{ root: classes.paper }}
+            variant="outlined"
+            style={{ marginBottom: 20 }}
+          >
+            <Grid container={true} spacing={3}>
+              <Grid item={true} xs={5}>
+                <Grid container={true} spacing={1}>
+                  <Grid item={true} xs={6}>
+                    <Typography variant="h3" gutterBottom={true}>
+                      {t_i18n('Name')}
+                    </Typography>
+                    <Tooltip title={work.name}>
+                      <Typography>
+                        {truncate(work.name, 80)}
+                      </Typography>
```

Comment:
```suggestion
                      <Typography sx={{ overflowX: 'hidden', textOverflow: 'ellipsis' }}>
                        {work.name}
                      </Typography>
```

We can use pure CSS to fix this issue without having to truncate the text arbitrarily

Reply:
Nice, and adding `whiteSpace: 'nowrap'` to display name in only 1 line to prevent too long names

Reply:
I'm not sure with this one. IMO there is no issue with the fact that the name is on multiple lines if it is long. I personally  prefer having the complete name on two lines rather than loosing information

Reply:
it's relevant. In the Acceptance Criteria, they ask for a tooltip in the name hover. So I thought there was a logic in not allowing the possibility of displaying a whole name. I'll check this with Nino

Reply:
I've agreed with him to display the full name directly, removing the tooltip which is no longer useful. Thanks for the feedback